public/javascripts/SVLabel/src/SVLabel/keyboard/Keyboard.js

```diff
@@ -47,11 +54,13 @@ function Keyboard (svl, canvas, contextMenu, googleMap, ribbon, zoomControl) {
      * @param degree
      */
     this._rotatePov = function (degree){
-        var heading =  svl.panorama.pov.heading;
-        var pitch = svl.panorama.pov.pitch;
-        var zoom = svl.panorama.pov.zoom;
-        heading = (heading + degree + 360) % 360;
-        svl.panorama.setPov({heading: heading, pitch: pitch, zoom: zoom});
+        if (!svl.map.getStatus("lockDisablePanning") && !svl.map.getStatus("disablePanning")){
```

Comment:
I think only the `disablePanning` should be sufficient. `lockDisablePanning` value is checked within disablePanning() method.